libcob/common.h

```diff
@@ -2850,4 +2850,31 @@ typedef	char *		cobchar_t;
 
 /*******************************/
 
+/************************/
+/* Functions in cconv.c */
+/************************/
+
+/* All supported conversions */
+enum ebcdic_table {
+	CB_EBCDIC_DEFAULT       = 0,
+	CB_EBCDIC_RESTRICTED_GC = 1,
+	CB_EBCDIC_IBM           = 2,
+	CB_EBCDIC_GCOS          = 3,
+};
+
+COB_EXPIMP const char *
+cob_get_collation_id_as_string (int col_id);
+
+COB_EXPIMP int
+cob_get_collation_by_id (int col_id,
```

Comment:
this function can be left as "internal" one (likely a s static function), but the external one should be `cob_get_collation_by_name` from below - which is then also used from cobc and can be used from GCSORT.